tap_kafka/sync.py

```diff
@@ -82,18 +84,38 @@ def init_kafka_consumer(kafka_config):
         # Non-configurable parameters
         'enable.auto.commit': False,
         'auto.offset.reset': 'earliest',
+        'value.deserializer': JSONSimpleDeserializer(),
     })
     consumer.subscribe([kafka_config['topic']])
 
     return consumer
 
 
+def get_timestamp_from_timestamp_tuple(kafka_ts):
```

Comment:
`message.timestamp()` is always a tuple as per the return type mentioned in the [doc](https://docs.confluent.io/5.5.2/clients/confluent-kafka-python/index.html?highlight=timestamp#confluent_kafka.Message.timestamp) 

So why check if it's a list?

Reply:
can we add type hints :) ?

Reply:
> So why check if it's a list?
That's only to make it testable with the current mechanism. The test messages are read from JSON files [here](https://github.com/transferwise/pipelinewise-tap-kafka/pull/49/files#diff-e471770fe9b7f052f26877b6a65cc3605a48dfc5868ce190038a97afc7d08aceR6). JSON doesn't know tuples only arrays. Would you prefer refactoring the test cases and convert the json arrays to tuples?

Reply:
fixed by 14d16d10bafd9ebab91526648aec3cb3302a8335